src/vuestic-theme/vuestic-components/vuestic-color-picker/VuesticAdvancedColorPicker.demo.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="row">
```

Comment:
Check demo styling example:
https://github.com/asvae/vue-book/blob/refactor/src/components/FileTree/BookComponentListItem.demo.vue

It's likely you don't need bootstrap here.